src/engine/SCons/Tool/ninja.py

```diff
@@ -0,0 +1,1396 @@
+# Copyright 2019 MongoDB Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Can we get this assigned to SCons and using MIT license?

Reply:
I can definitely reach out to our legal about getting this (and other MongoDB SCons tools) over to MIT.

Reply:
FYI this relicense was completed: https://github.com/mongodb/mongo/commit/77162e6d10c23eb05e8805290ac8dc0b579f04a7

Also please note that the newest version of the MongoDB ninja tool is here: https://github.com/mongodb/mongo/blob/master/site_scons/site_tools/ninja_next.py

There are still a few small mongodb-isms in there, but we will be working to scrub them out. Some of the recent issues that I've filed are related to doing so.

Reply:
File name update: [https://github.com/mongodb/mongo/blob/master/site_scons/site_tools/ninja.py](https://github.com/mongodb/mongo/blob/master/site_scons/site_tools/ninja.py)

Reply:
> File name update: https://github.com/mongodb/mongo/blob/master/site_scons/site_tools/ninja.py

This PR is closed and the work has been moved to PR #3642 please post your comment there and any further comments.